src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cl from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const actualTabId = tabs.some(tab => selectedTabId === tab.id)
+    ? selectedTabId
+    : tabs[0].id;
```

Comment:
Better create a variable for selectedTab and set the first tab as the default value

Reply:
What do you mean behind this? Just for context, we have one test that provides an incorrect ID, and this check is merely an attempt to prevent that.